lib/stack_master/template_compilers/sparkle_formation.rb

```diff
@@ -1,18 +1,52 @@
+require_relative '../sparkle_formation/compile_time/parameters_validator'
+require_relative '../sparkle_formation/compile_time/definitions_validator'
+require_relative '../sparkle_formation/compile_time/state_builder'
+
```

Comment:
This can be changed to `require 'stack_master/compile_time/parameters_validation'`

Reply:
Note: I noticed the use of `autoload`. This is apparently deprecated and should not be used anymore. [see](http://www.rubyinside.com/last-weeks-top-ruby-news-rails-3-1-3-autoload-deprecated-and-conferences-5650.html)

Reply:
That's a long way off for now

Reply:
There are 21 other `require_relative` calls, can we get rid of them too?